bamboo-server/bamboo-server.sh

```diff
@@ -19,6 +19,14 @@ else
   echo "-> Extracting to $BAMBOO_DIR ..."
   tar xzf /tmp/atlassian-bamboo.tar.gz -C /opt
   rm -f /tmp/atlassian-bamboo.tar.gz
+  
+  echo "Add MySQL"
```

Comment:
Not everyone will want to use MySQL and need this connector. I'd add a BAMBOO_DB env var in the Dockerfile, defaulting to embedded but also supporting mysql and postgresql. You could then check that and see if it's appropriate to download a particular driver.